server/oracleService.js

```diff
@@ -30,55 +31,63 @@ module.exports = class OracleService {
       oracleTtl: {type: 'delta', value: this.ttl}
     });
 
+    logger.info("oracle id", this.oracle.id);
+
     if (this.autoExtend) {
       this.extendIfNeeded();
       this.extendIfNeededInterval = setInterval(() => {
         this.extendIfNeeded();
-      }, (this.ttl / 5) * 3 * 60 * 1000); // every ttl/5 blocks
+      }, (this.ttl / 5) * (60 / 3) * 1000); // every ttl/5 blocks
     }
-    console.log("Oracle Id", this.oracle.id);
   };
 
   extendIfNeeded = async () => {
+    logger.debug("checking to extend oracle");
     const height = await this.aeternity.client.height();
 
     if (height > this.oracle.ttl - this.ttl / 5) {
       this.oracle = await this.oracle.extendOracle({type: 'delta', value: this.ttl});
-      console.log("Extended Oracle at height:", height, "new ttl:", this.oracle.ttl);
+      logger.debug("extended oracle at height:", height, "new ttl:", this.oracle.ttl);
```

Comment:
shouldnt this be a higher level than debug?

Reply:
It can be